### ISSUE TITLE
[cloud-tensorflow-mnist]: Notebook return error on step 'Let's have a look at the data'

### ISSUE THREAD

**Issue by vvillarreal-cfee** [googlecodelabs/feedback#1451]
The notebook returns an error when try to use the display_digits() because the parameter b of grid was deprecated in [matplotlib 3.5](https://matplotlib.org/stable/api/prev_api_changes/api_changes_3.5.0.html#the-first-parameter-of-axes-grid-and-axis-grid-has-been-renamed-to-visible), and renamed visible.

Reference: https://github.com/matplotlib/matplotlib/issues/25267#issuecomment-1437368645

**Comment by vvillarreal-cfee** [comment]
To fix this, need to change the line:

```
plt.grid(b=None)
```
to:
```
plt.grid(visible=None)
```
inside the display_digits() function definition, on the `keras_01_mnist.ipynb` notebook.
